app/queries/intent_metrics_query.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+# Query to return the metrics on the intent selections of the students in a
+# specific draw.
+class IntentMetricsQuery
+  class << self
+    delegate :call, to: :new
+  end
```

Comment:
I'm pretty sure I took it from Reservations, but maybe you haven't dealt much with query objects there. It's really only used to turn a query object into a scope more easily (pass the class instead of an instance of the class), and since this is more of a standalone object I think I'm actually going to take it out.

Thanks for calling out all the missing documentation 😄.